rpcs3/Emu/Cell/PPUThread.cpp

```diff
@@ -4523,10 +4523,13 @@ bool ppu_initialize(const ppu_module& info, bool check_only)
 		// Check object file
 		if (jit_compiler::check(cache_path + obj_name))
 		{
-			if (!jit && !check_only)
```

Comment:
Why was this changed?

Reply:
It was a bug, g_progr_ptotal is incremented if check_only is false. It does not care about null jit.

Reply:
Actually, this change is wrong.
It's incremented later again if it needs to be compiled a couple of lines below.

So you're incrementing the uncached ones twice now.